index.js

```diff
@@ -55,9 +40,16 @@ module.exports = {
     // allow dangling underscores in variable names (for "private" class methods)
     'no-underscore-dangle': 0,
 
-    // disallow spaces in between curly braces
-    'babel/object-curly-spacing': [2, 'never'],
-    'object-curly-spacing': 0,
+    // allow event listeners on static elements (e.g. onClick on divs)
+    'jsx-a11y/no-static-element-interactions': 0,
+
+    'prettier/prettier': [2, {
+      printWidth: 100,
+      semi: false,
+      singleQuote: true,
+      trailingComma: 'all',
```

Comment:
`es5` would be more consistent with what we currently do (`'all'` adds commas in function args too)